src/server/main_service.cc

```diff
@@ -2683,7 +2698,9 @@ constexpr uint32_t kWatch = FAST | TRANSACTION;
 constexpr uint32_t kUnwatch = FAST | TRANSACTION;
 constexpr uint32_t kDiscard = FAST | TRANSACTION;
 constexpr uint32_t kEval = SLOW | SCRIPTING;
+constexpr uint32_t kEvalRo = SLOW | SCRIPTING;
```

Comment:
@kostasrim should we add `READ` here as well? (and below)

Reply:
I was about to say yes but I double checked and you would be surprised: `@slow, @scripting` are the only tags for this command but not `READ` :exploding_head:

Reply:
Is there any harm in adding `READ` here though?

Reply:
well we are breaking the spec if we do :shrug:

Reply:
Just to check, so leaving this as is? (i.e. not adding `READ`?)

Reply:
Yes, that should make us compatible with Valkey I guess, and we can always relax this without breaking any existing user (which is not true for the opposite case)